demos/companion-app/tailwind.config.ts

```diff
@@ -56,10 +56,6 @@ module.exports = {
         md: "calc(var(--radius) - 2px)",
         sm: "calc(var(--radius) - 4px)",
       },
-      fontFamily: {
-        sans: ["var(--font-geist-sans)"],
-        mono: ["var(--font-geist-mono)"],
-      },
```

Comment:
Add back the font please.

Reply:
Fixed. Had to add the `geist` font dependency also and configure the app layout with the font.